rust/otap-dataflow/crates/engine/src/control.rs

```diff
@@ -32,6 +32,12 @@ pub enum NodeControlMsg {
         id: u64,
         /// Human-readable reason for the NACK.
         reason: String,
+
+        /// Placeholder for optional return value, making it possible for the
+        /// retry sender to be stateless.
+        ///
+        /// TODO: should this be boxed to reduce the size of NodeControlMsg?
+        rvalue: Option<PData>,
```

Comment:
Any particular reason not naming this field pdata?

Reply:
Renamed `pdata`.